<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorprofilepage</class>
 <widget class="QWidget" name="doctorprofilepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>User ID :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Departement :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PHOTO</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPhoto">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Personal Information :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDepartement">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePersonalInformation">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangePhoto">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>280</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Photo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>530</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbFee">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>74</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbDailyLimit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dteWorkTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>209</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Work Time :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Appointment Fee :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Daily Limit Patient :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
